README.md

```diff
@@ -6,8 +6,8 @@ The Universal Device Management Interface (UDMI) provides a high-level specifica
 management and operation of physical IoT systems. This data is typically exchanged
 with a cloud entity that can maintain a "digital twin" or "shadow device" in the cloud.
 
-* [Core UDMI documentation](http://faucetsdn.github.io/udmi/docs/) for tools and specifications
-* [Message schema definition](https://github.com/faucetsdn/udmi/tree/master/schema) with ([_🧬Interactive Viewer_](http://faucetsdn.github.io/udmi/gencode/docs/))
```

Comment:
The interactive viewer link on the home page should be to http://faucetsdn.github.io/udmi/ otherwise it's unrendered HTML code. Or a big "Read Docs" button which links to http://faucetsdn.github.io/udmi.